### PR TITLE
ENCD-4718-4884 Deployment updates

### PR DESCRIPTION
**Pull request by caseylitton**
This pr covers two jira tickets.  This ticket should be merged and not squash merged.

### CODE REVIEW

File: cloud-config/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+* Development Deployment or Demo: Single instance with front, postgres, es data, es head 
+* Elect Cluster: Single instance with frontend and postgres.  5 Node cluster with self elected head node.
+* Wait Cluster: Single instance with frontend and postgres.  5 Node cluster with 6th instance for head node.
+
+## Dev Deployment
+    * Made       Prebuild: $bin/deploy -n dev-conf --save-config-name 20191002
+        # Wrote new config to ./cloud-config/prebuilt-config-yamls/20191002-demo.yml
+    * Diff Build/Prebuild: $bin/deploy -n dev-conf --use-prebuilt-config 20191002-demo --diff-configs
+    * Deploy     Prebuild: $bin/deploy -n dev-conf --use-prebuilt-config 20191002-demo
+    * Deploy        Build: $bin/deploy -n dev-conf
```
Comment (hitz) [inline cloud-config/README.md]: should -n dev-conf be the default?

Comment (caseylitton) [reply cloud-config/README.md]: I'm not trying to explain the deploy command in this readme.  I think people understand it well enough anyway.  Or get errors for trying to create an instance with the same name!

Comment (caseylitton) [reply cloud-config/README.md]: I fixed the ticket number name also.

File: cloud-config/deploy-run-scripts/encd-install.sh [hunk 2]
```diff
@@ -23,12 +24,14 @@ until sudo -u postgres psql postgres -c ""; do sleep 10; done
 sudo -u encoded sh -c 'cat /dev/urandom | head -c 256 | base64 > session-secret.b64'
 sudo -u encoded bin/create-mapping production.ini --app-name app
 sudo -u encoded bin/index-annotations production.ini --app-name app
-if test "%(REGION_INDEX)s" = "False"
+cp /srv/encoded/etc/logging-apache.conf /etc/apache2/conf-available/logging.conf
+# Create encoded apache conf
+sudo -u root cat etc/conf-apache-parts/head.conf > /etc/apache2/sites-available/encoded.conf
+sudo -u root sed "s/APP_WORKERS/$APP_WORKERS/" <  etc/conf-apache-parts/app.conf > /etc/apache2/sites-available/encoded.conf
```
Comment (caseylitton) [inline cloud-config/deploy-run-scripts/encd-install.sh]: This line and all the rest should be appending

**Pull request merged**